server/chat-plugins/uno.ts

```diff
@@ -246,8 +246,11 @@ export class UNO extends Rooms.RoomGame {
 				this.sendToRoom(`|raw|${Utils.escapeHTML(name)} has not picked a color, the color will stay as <span style="color: ${textColors[this.topCard.changedColor]}">${this.topCard.changedColor}</span>.`);
 			}
 		}
-		if (this.awaitUno === userid) this.awaitUno = null;
-
+		if (this.awaitUno === userid) this.awaitUno = null; {
```

Comment:
uh, what? Does this work locally?

Reply:
For me yes:
Now: http://prntscr.com/yvvqsd
Before: http://prntscr.com/yvvu2v

Reply:
Okay then.
```suggestion
		if (this.awaitUno === userid) this.awaitUno = null;
```